examples/simultaneous_translation/docs/baseline.md

```diff
@@ -134,7 +134,8 @@ CUDA_VISIBLE_DEVICES=1 python $FAIRSEQ/train.py data-bin/mustc_en_de \
     --log-format json \
     --log-interval 10 \
     --criterion cross_entropy_acc \
-    --user-dir $FAIRSEQ/examples/simultaneous_translation
+    --user-dir $FAIRSEQ/examples/simultaneous_translation \
+    --online-features
```

Comment:
What's the default?

Reply:
The default is to read offline features from disk, this flag will do online extraction. Or is it better to have offline as default?

Reply:
If offline is faster, it should be a good default.